README.md

```diff
@@ -44,3 +44,82 @@ along with this program.  If not, see https://www.gnu.org/licenses/
 ___
 
 Project Website: https://metrocs.github.io/imagelab/
+
+## Build Instructions 
+
+BUILD INSTRUCTIONS
+
+◾️Introduction
+
+This documents intent is to support developers in understanding how to 
+build the Image Lab project. This document offers support on key useful commands and error solutions that can make building Image Lab a smoother process.
+
+◾️Before You Build
+- Install Gradle
+ sudo apt install gradle
+- Install Canberra
+ Sudo apt install libcanberra-gtk-module libcanberra-gtk3-module
+
+
+◾️What is Gradle
+
+Gradle is a automated build system that makes working with large projects easier.
+It streams line the build process by skipping re-execution and allows developers to preform tasks suing one line of code in the command line.
+
+
+
+◾️How does Gradle Work?
+
+▪️Project 
+- A project is a depiction of what needs to be built and how we get an artifact at the end of a build.
+
+settings.gradle - Where projects are registered.
+build.gradle - constructs commands that can be used to execute tasks.
+
+
+▪️Task
+-A task is used to execute a action during the build process. For example, ./gradlew compileJava will compile the main class in the build. 
+
+
+
+▪️3 Build Phases
+
+The Initialization Phase 
+- In this phase Gradle tires to establish what projects are in the build process.
+Gradle will look to the file setting.gradle to identify the project or projects. 
+
+The Configuration Phase
+- In this phase Gradle locates and executes build scripts for each project. This would be in our case the build.gradle file. Understand that when the build script is executed the tasks inside the build script are not. From this Gradle uses an algorithm called DAG that creates a sort of map to what dependents are needed for each task. In short this allows us to use gradle commands.
+
+The Execution Phase
+- In this phase Gradle locates what tasks are dependent on each other, and is able to actually start the build process. This is where we are able to execute tasks in the command line in order to clean, compile, run, and use other commands to build the project. 
+
+
+
+
+◾️Possible Errors and Solutions During Build
+Error
+-Failed to load module "canberra-gtk-module"
+Solution
+-Sudo apt install libcanberra-gtk-module libcanberra-gtk3-module
```

Comment:
Why would someone get this error?  This seems to apply to particular, _non-required_ tools and operating systems.